### ISSUE TITLE
Database copy of the data

### ISSUE THREAD

**Issue by samin-batra** [jolpica/jolpica-f1#116]
Hi,

I might be missing something here but when I set-up the jolipca-f1 API using the devcontainer and ran the steps to import data, I cannot find where to access the database to run some queries from my side. Is there a way to do so? In the documentation it mostly mentions about running the API and accessing the data through that.

Thank You,

**Comment by brunocassani** [comment]
Do you have the `DATABASE_SECRET_URL`? Using `echo` from the command line worked for me.

**Comment by samin-batra** [comment]
No, when I used the echo command, it returned empty. However, I have been able to connect to postgres by using the localhost URL. It worked. Thanks!

**Comment by williamsdb** [comment]
I came here looking for database dumps as were previously provided by Ergast which [my website](https://williamsdb.com) relies on and found this thread. Seeing that Jolpica doesn't provided dumps I have written a routine that uses the api to retrieve data and then inserts it into an existing Ergast database. You can [find the code here](https://github.com/williamsdb/jolpica2Ergast). Hopefully it will be of use to others too.

**Comment by Vucko95** [comment]
Thanks @williamsdb I also also had to write my own, but I will try this one as well when I have time, as for me only purpose of Ergast was database dumps, as surely I wanna write my own logic for other stuff.

**Comment by jolpica** [comment]
Closing in favour of #154
